docs/quickstarts/nextjs.mdx

```diff
@@ -82,8 +82,9 @@ The final result should resemble the following:
 
 [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware#clerk-middleware) grants you access to user authentication state throughout your application, on any route or page. It also allows you to protect specific routes from unauthenticated users. To add `clerkMiddleware()` to your application, follow these steps:
 
-
-1. Create a `middleware.ts` file. Your `middleware.ts` file should be placed inside the root directory alongside `.env.local`, or inside the `src/` directory if you are using it.
+1. Create a `middleware.ts` file. The location of the middleware depends on if you are using the `/src` directory in your project:

```

Comment:
```suggestion
1. Create a `middleware.ts` file. 

```

I feel like removing this will make the section look a little cleaner. With this line and the bullet points it feels a little messy.